rosbag2_transport/src/rosbag2_transport/bag_rewrite.cpp

```diff
@@ -122,6 +122,7 @@ setup_topic_filtering(
       std::stringstream qos_profiles;
       qos_profiles << input_topics_qos_profiles[topic_name];
       topic_metadata.offered_qos_profiles = qos_profiles.str();
+      // TODO(morlov:) Get message definition for topic from input storage
       writer->create_topic(topic_metadata);
```

Comment:
Nitpick. I would prefer to use `writer->create_topic(topic_metadata, {});` here to show that we need to use here version with message definition.